openquake/calculators/event_based_risk.py

```diff
@@ -179,6 +173,12 @@ def zeroN():
                         monitor.ela_dt))
 
             # agg losses
+            glosses = out.event_loss_per_asset.sum(axis=1)
+            ilosses = out.insured_loss_per_asset.sum(axis=1)
+            agglosses = {
+                rupid: numpy.array([gloss, iloss], F32)
+                for rupid, gloss, iloss in zip(out.rupids, glosses, ilosses)
```

Comment:
The same thing is called in a number of different ways (`all_losses, event_loss_per_asset, groundloss, glosses`). Perhaps we should attempt to unify them.